ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -1039,7 +1039,7 @@ public static List<String> mergeUniqElems(List<String> src, List<String> dest) {
     return src;
   }
 
-  private static final String tmpPrefix = "_tmp.";
+  private static final String tmpPrefix = "-tmp.";
   private static final String taskTmpPrefix = "_task_tmp.";
```

Comment:
Also, what will happen after Hive upgrade? Would not be something incompatible? IE: Remaning temp file?

Reply:
The tmp files are present in the staging dir hence they should deleted after query execution.